.emacs.d/lisp/code/languages/web-c.el

```diff
@@ -2,6 +2,7 @@
 (add-to-list 'auto-mode-alist '("\\.scss\\'" . web-mode))
 (add-to-list 'auto-mode-alist '("\\.sass\\'" . web-mode))
 (add-to-list 'auto-mode-alist '("\\.css\\'" . web-mode))
+(add-to-list 'auto-mode-alist '("\\.mjs\\'" . web-mode))
 (add-to-list 'auto-mode-alist '("\\.jsx?\\'" . web-mode))
```

Comment:
is .mjsx a thing? if so we can combine

Reply:
Oh true.

Do you mean something like this?

```suggestion
(add-to-list 'auto-mode-alist '("\\.m?jsx?\\'" . web-mode))
```

Which would obviously replace the other two… so this?

```diff
- (add-to-list 'auto-mode-alist '("\\.mjs\\'" . web-mode))
- (add-to-list 'auto-mode-alist '("\\.jsx?\\'" . web-mode))
+ (add-to-list 'auto-mode-alist '("\\.m?jsx?\\'" . web-mode))

Reply:
yeah but only if .msjx is legit. if not, what you have is good

Reply:
It's definitely a rare thing if it is. I think we'll leave it as is for now.